calrissian/job.py

```diff
@@ -119,6 +119,10 @@ def add_persistent_volume_entries_from_pod(self, pod):
         inspector = KubernetesPodVolumeInspector(pod)
         for mount_path, sub_path, claim_name, read_only in inspector.get_mounted_persistent_volumes():
             self.add_persistent_volume_entry(mount_path, sub_path, claim_name, read_only)
+            # If the mount path is for a workspace, mount the volume to the workspace-data directory
+            if mount_path.startswith("/workspaces"):
+                log.info("Adding volume mount for workspace-data directory")
+                self.add_volume_binding(mount_path, "/workspace-data", False)
```

Comment:
On mounting to fixed path /workspace-data:
Could be a problem if it's possible to have two EFS directories in a workspace (I don't -think- this is possible though?)
Also will be a problem if the stage in does use a different workspaces EFS (will mount to the same place). This could be sorted out later though.
Thoughts?

Reply:
Good point, perhaps we rename this to be `/workspace-data/stage-in` or something like that? Then the other _processing_ volumes can be mounted at `/workspace-data/processing`? Is that clear? 
Otherwise, we could just name the directory after each workspace, so we have `/workspace-data/own-workspace` and then `/workspace-data/other-workspace`. Open to other suggestions, please

Reply:
Updated to be identical to the parent mount path, e.g. `workspaces/<workspace-name>`.

Reply:
Another update to ensure all volume mounts from parent are mounted here too, except `calrissian-wdir` which is already mounted.